pkg/mock_server_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"log"
 	"net"
 
+	"github.com/golang/protobuf/ptypes/empty"
 	. "github.com/open-traffic-generator/openapiart/pkg/sanity"
```

Comment:
do not use "."

prepend "sanity." to all the structs/funcs that are imported from "sanity".

Reply:
updated